packages/cosmic-proto/proto/icq/v1/genesis.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
```

Comment:
please add a README with where these types came from and how to update

consider referencing https://github.com/Agoric/agoric-sdk/issues/9018

Reply:
Please let me know if this is what you're looking for: [5e6252b](https://github.com/Agoric/agoric-sdk/pull/9198/commits/5e6252b173aaf51b231fb94f00b3fa834c51a4ec)

If not, more specific direction is appreciated.